common/test/acceptance/pages/studio/library.py

```diff
@@ -168,6 +169,7 @@ def count(self, count):
         Sets value of children count input
         """
         count_text = self.get_setting_element(self.COUNT_LABEL)
+        count_text.send_keys(Keys.BACK_SPACE)
```

Comment:
What if the default value was 10? Would hitting backspace once just delete the "0" and leave the "1"?
Perhaps another solution like home, shift-end, delete would work better?